lib/datastore/transaction.js

```diff
@@ -119,12 +123,10 @@ Transaction.prototype.begin = function(callback) {
  * @param {function} callback - The callback function.
  *
  * @example
- * transaction.begin(function(err) {
- *   transaction.rollback(function(err) {
- *     if (err) {
- *       // Transaction could not be rolled back.
- *     }
- *   });
+ * transaction.rollback(function(err) {
```

Comment:
Just cuz I don't know this issue on hand and like you said, doesn't really belong in this PR.